webapi/src/main/scala/org/knora/webapi/store/triplestore/api/TriplestoreService.scala

```diff
@@ -211,3 +153,36 @@ trait TriplestoreService {
   def insertDataGraphRequest(graphContent: String, graphName: String): Task[Unit]
 
 }
+
+object TriplestoreService {
+  object Queries {
+
+    sealed trait SparqlQuery { val sparql: String }
+
+    case class Ask(sparql: String) extends SparqlQuery
+    object Ask {
+      def apply(sparql: TxtFormat.Appendable): Ask = Ask(sparql.toString)
+    }
+
+    case class Select(sparql: String, isGravsearch: Boolean) extends SparqlQuery
+    object Select {
+      def apply(sparql: TxtFormat.Appendable, isGravSearch: Boolean = false): Select =
+        Select(sparql.toString, isGravSearch)
+
+      def apply(sparql: String): Select = Select(sparql, isGravsearch = false)
+    }
+
+    case class Construct(sparql: String, isGravsearch: Boolean) extends SparqlQuery
+    object Construct {
+      def apply(sparql: TxtFormat.Appendable, isGravSearch: Boolean = false): Construct =
+        Construct(sparql.toString, isGravSearch)
+
+      def apply(sparql: String): Construct = Construct(sparql, isGravsearch = false)
+    }
```

Comment:
```suggestion
    case class Select(sparql: String, isGravsearch: Boolean) extends SparqlQuery
    object Select {
      def apply(sparql: TxtFormat.Appendable, isGravsearch: Boolean = false): Select =
        Select(sparql.toString, isGravsearch)

      def apply(sparql: String): Select = Select(sparql, isGravsearch = false)
    }

    case class Construct(sparql: String, isGravsearch: Boolean) extends SparqlQuery
    object Construct {
      def apply(sparql: TxtFormat.Appendable, isGravsearch: Boolean = false): Construct =
        Construct(sparql.toString, isGravsearch)

      def apply(sparql: String): Construct = Construct(sparql, isGravsearch = false)
    }
```
or is it by purpose?